.github/workflows/ci.yml

```
@@ -19,12 +19,12 @@ jobs:
 
     runs-on: ubuntu-latest
     container:
-      image: rootproject/root:6.24.06-ubuntu20.04
+      image: rootproject/root:6.26.02-ubuntu20.04
```

```suggestion
      image: rootproject/root:6.24.06-ubuntu20.04
```

Just in case this is the problem.